source/state_representation/README.md

```diff
@@ -6,6 +6,16 @@ Those are a set of helpers functions to handle common concepts in robotics such
 the link between them and the robot state.
 This description covers most of the functionalities starting from the spatial transformations.
 
+Table of contents:
+* [Cartesian state](#cartesian-state)
+  * [Cartesian state operations](#cartesian-state-operations)
+  * [Changing of reference frame](#changing-of-reference-frame)
+  * [Specific state variables](#specific-state-variables)
+  * [Conversion between Cartesian state variables](#conversion-between-cartesian-state-variables)
+* [Joint state](#joint-state)
+  * [Joint state operations](#joint-state-operations)
+  * [Conversion between Cartesian state variables](#conversion-between-joint-state-variables)
```

Comment:
Last link text here is still "Cartesian" but should be "joint" (I'm suggesting lowercase because either you refer to the class type JointState with upper case or to the regular name "joint state". Cartesian is always capital because it's a proper noun of course)

I tested all the local links by viewing this revision in the rendered markdown format, and they all link correctly 👍

Reply:
yes you are completely right

Reply:
oh yeah Cartesian should not be made lowercase... damn it